appinfo/info.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<info>
+	<id>brute_force_protection</id>
+	<name>Brute Force Protection</name>
+	<description>ownCloud brute force protection app. It hardens ownCloud security by implementing brute force protection.</description>

```

Comment:
@pmaier1 please review - THX

Reply:
I would change the summary and description texts a little if @karakayasemi is fine with it.

Reply:
Any suggestion helps me a lot. I am totally fine with it.